knowledge-base/user-guides/view_only.md

```diff
@@ -12,12 +12,25 @@ attribution: "<!-- Icon is based on work by Freepik (http://www.freepik.com) and
 A view-only wallet can only see which incoming transactions belong to you. It can not spend any of your Monero, in fact it can't even see outgoing transactions from this wallet. This makes view-only wallets particularly interesting for
 
 * Developers writing libraries to validate payments
-* End users validating incoming transactions to cold wallets
+* End users validating incoming transactions to cold wallets.
 
-### Creating A View-Only Wallet
+### Creating and Using A View-Only Wallet
 
-You can create a view-only wallet from any existing wallet.
+You can create a view-only wallet from any existing wallet. We'll call the original wallet OfflineWallet and the view-only wallet ViewWallet.
 
-Open an existing wallet or create a new one using `monero-wallet-cli`. In the wallet, type `address` and `viewkey` to display the wallet's address and secret viewkey. Type `exit` to close the wallet.
+* Open an existing wallet or create a new one using `monero-wallet-cli`. That will be the OfflineWallet.
+* In the wallet, type `address` and `viewkey` to display the wallet's address and secret viewkey. Type `exit` to close the wallet.
+* Next, create your view-only wallet by typing `monero-wallet-cli --generate-from-view-key wallet-name`. The last argument will be your new wallet's file name. You will be prompted for `Standard address` and `View key` by the wallet. Paste in your original wallet's address and secret view key. Next, enter and confirm a password for your new wallet and you're done. 
+You will now have an additional wallet, a view-only wallet with its separate password. 
+We'll call the new wallet is ViewWallet.
+* Finally, move the three Viewwallet files (viewWallet, viewWallet.address and viewWallet.KEYS) to an online computer.
 
-Next, create your view-only wallet by typing `monero-wallet-cli --generate-from-view-key wallet-name`. The last argument will be your new wallet's file name. You will be prompted for `Standard address` and `View key` by the wallet. Paste in your original wallet's address and secret view key. Next, enter and confirm a password for your new wallet and you're done.
+### Creating, signing and broadcasting transctions using ViewWallet and OfflineWallet
+
+To create a transaction from the viewWallet, start with the ViewWallet: 
+* Run monero-wallet-cli on the online computer and load the ViewWallet.
+* Next, use the monero-wallet-cli command `transfer` with parameters ADDRESS and AMOUNT. The viewWallet will create a new file in its directory called `unsigned-monero-transaction`. 
+* Copy that file to an SD card/USB key and move it to the directory of the OfflineWallet. For extra protection, use an SD card for that purpose only.
```

Comment:
> For extra protection, use an SD card for that purpose only.

Seems you are suggesting that they should not use the SD card or USB key for any other purpose?